### ISSUE TITLE
Achieving reported accuracy at local

### ISSUE THREAD

**Issue by LLYXC** [EIDOSlab/entangling-disentangling-bias#1]
Dear author,

Thank you so much for generously sharing the code. I am trying to follow your study at my local computer. However, when I tried the code with colour_mnist, with alpha and beta all set to 0.1, my unbiased accuracy was around 0.77 after convergence.

I noticed that you have used wandb for hyper-parameters tuning. Is that a must? Can I also reach similar results with running and tuning on local? Would you mind sharing some experience on the hyper-parameters tuning?

Thank you very much!

**Comment by carloalbertobarbano** [comment]
Hi. Yes, in order to reach the high accuracy reported in the paper, some kind of hyperparameter tuning is necessary ..wandb is not a must (though it's already implemented), you can use any toolkit you prefer such as raytune etc. 
The code shouldn't be too hard to adapt. 

If you want to skip tuning altogether, a general rule of thumb for a "safer" hyperparams choice is having a larger beta and a smaller alpha. 
If you cannot run an exhaustive search due to hardware constraint, consider employing an early terminate techinque (e.g. if the validation accuracy has not significantly increased after N epochs.

Anyways, I can confirm that with ```alpha = beta = 0.1``` i obtain a final test accuracy of ```0.7795``` on ```rho=0.997```

**Comment by LLYXC** [comment]
Thank you so much for the quick and kind reply! The test accuracy under `alpha = beta = 0.1` on `rho=0.997` is the same with my local results. I shall further try hyper-parameters tuning strategies :-)
